framework/src/main/java/org/tron/common/application/ApplicationImpl.java

```diff
@@ -75,7 +75,7 @@ public void shutdown() {
     dbManager.stopRePushThread();
     dbManager.stopRePushTriggerThread();
     EventPluginLoader.getInstance().stopPlugin();
-    logger.info("******** end to shutdown ********");
+    logger.info("******** stop to shutdown ********");
```

Comment:
It seems that this got the wrong meaning.

"stop to shutdown" = "stop something (which is happening now) and then shutdown"

Reply:
ok, got it, will change